govgen-1/README.md

```diff
@@ -41,13 +38,29 @@ govgend gentx your-key-name 1000000ugovgen \
   --security-contact "security@foo.network"
 ```
 
-## Node operator recommendation
+## Operate the node 
+
+### Genesis file [download link](https://atomone.fra1.digitaloceanspaces.com/govgen/govgen-1/genesis.json)
+
+The final genesis is avalable at [this link](https://atomone.fra1.digitaloceanspaces.com/govgen/govgen-1/genesis.json).
+
+```sh
+$ wget -O ~/.govgen/config/genesis.json https://atomone.fra1.digitaloceanspaces.com/govgen/govgen-1/genesis.json
+$ md5sum ~/.govgen/config/genesis.json
+53800c74844af747b1032ef6c5bf3fc4  genesis.json
+```
+
+### Recommendations
+
+| minimum-gas-prices | 0.001ugovgen |
+|--------------------|--------------|
```

Comment:
let's also add a reference to remind people to look at seed.txt and peers.txt for seeds and peers